gcp/v2/README.md

```diff
@@ -0,0 +1,83 @@
+# Kubeflow on KCC Installation Guide
```

Comment:
Should we mark this as experimental or Alpha?

Reply:
+1. I'm wondering why this new architecture is in a directory named **v2** - is it destined for a v2 release of Kubeflow? Perhaps the experimental/alpha label can explain that too.